embedded-google-storage/src/main/java/com/playtika/test/storage/StorageResourcesGenerator.java

```diff
@@ -0,0 +1,47 @@
+package com.playtika.test.storage;
+
+import com.google.cloud.NoCredentials;
+import com.google.cloud.storage.BucketInfo;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.annotation.PostConstruct;
+
+import java.util.Collection;
+
+@Slf4j
+public class StorageResourcesGenerator {
+
+    private final Storage storage;
+    private final Collection<String> buckets;
+
+
+    StorageResourcesGenerator(String storageHostUrl, StorageProperties storageProperties) {
+        this.storage = buildStorageService(storageHostUrl, storageProperties.getProjectId());
+        this.buckets = storageProperties.getBuckets();
+    }
+
+    private Storage buildStorageService(String storageHostUrl, String projectId) {
+        return StorageOptions.newBuilder()
+            .setHost(storageHostUrl)
+            .setProjectId(projectId)
+            .setCredentials(NoCredentials.getInstance())
+            .build()
+            .getService();
+    }
+
+
+    @PostConstruct
+    protected void init() {
+        log.info("Creating buckets.");
+        buckets.forEach(this::createBucket);
```

Comment:
is it possible to create bucket via cli, so that we do not have dependency on `google-cloud-storage`?

Reply:
i've just looked into the transitive dependencies of `google-cloud-storage` and it looks like it comes with half of the internet :), so if possible please use rest api. for bucket creation, i suppose, this won't be difficult?
https://cloud.google.com/storage/docs/json_api/v1/buckets/insert